sentry-android-core/src/main/java/io/sentry/android/core/AndroidContinuousProfiler.java

```diff
@@ -157,6 +189,28 @@ public synchronized void close() {
     stop();
   }
 
+  private void sendChunks(final @NotNull IScopes scopes, final @NotNull SentryOptions options) {
+    try {
+      options
+          .getExecutorService()
+          .submit(
+              () -> {
+                final ArrayList<ProfileChunk> payloads = new ArrayList<>(payloadBuilders.size());
+                synchronized (payloadBuilders) {
+                  for (ProfileChunk.Builder builder : payloadBuilders) {
+                    payloads.add(builder.build(options));
+                  }
+                  payloadBuilders.clear();
+                }
+                for (ProfileChunk payload : payloads) {
+                  scopes.captureProfileChunk(payload);
```

Comment:
how many pending chunks we could have potentially? Just thinking if we burst the backend here and cause rate-limit to kick in? Maybe we could add a short 100ms delay between sending chunks if we expect a few.

Reply:
i expect only one, honestly
The only case when we can have more is app start profiling, but i'd expect one there, too, most of the times